core/biogeochem/casa_cnp.F90

```diff
@@ -1360,8 +1360,8 @@ SUBROUTINE casa_delplant(veg, casabiome, casapool, casaflux, casamet, &
         do nP=1, mplant
            casaflux%FluxFromPtoCO2(npt,nP) = &
                 (1.0_r_2-casaflux%kplant(npt,nP)) * casaflux%kplant_fire(npt,nP) * &
-                casapool%cplant(npt,nP) * &
-                (1.0_r_2 - sum(casaflux%fromPtoL_fire(npt,:,nP)))
+                casapool%cplant(npt,nP) !CLN *  & Shouldn't it be min(1-casaflux%kplant,X)???
```

Comment:
we need to check this - it maybe undoing something that we need to retain for BLAZE.

Reply:
Revision history suggests that LArs/Yohanna have updated NESP2pt9_BLAZE after BLAZE_9184 split.  Assuming that Merge_Spin_1951 came from NESP2pt9 then we should accept this (and the other BLAZE related edits.

Reply:
I branched from NESP2pt_BLAZE a few weeks ago to get local changes into the repository.  My local changes were made on code base that came from the NESP2pt_BLAZE branch in svn in around July 2023.

Reply:
Can you please advise if which code is correct?

Reply:
Again I think this is a change implemented by Lars post the branching off of BLAZE_9184.  If true then this should be retained.

Perhaps need to contact Lars directly on these two.

Reply:
We could ask him to comment directly on the issue. Do you know what Lars' is name here?

Reply:
Given the timeline of revisions we should retain this change in the merge between spin_1951 and BLAZE_9184.  The missing step was to update BLAZE_9184 prior to the svn closing.